mitodl/mitxpro
# feat: webinars

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes https://github.com/mitodl/mitxpro/issues/2615

#### What's this PR do?
Adds a dedicated page for webinars.

- Admins can add webinars in CMS.
- Users can view webinars at `/webinars/`

#### How should this be manually tested?

- Visit CMS and add a `Webinar Index Page` as a child page for your home page. This will hold all the webinars.
- Add some `Webinar` child pages for `Webinar Index Page`
- Visit `/webinars/` and verify that the webinars are visible.
- Verify the design

NOTE: Delete Webinar Index Page before **unapplying** migrations.

#### Where should the reviewer start?
By adding some webinar pages in the CMS

#### Screenshots (if appropriate)
CMS:
<img width="819" alt="Screen Shot 2023-04-18 at 9 14 28 PM" src="https://user-images.githubusercontent.com/52656433/232839746-13b56fc7-88e2-4f6a-b122-ea6444558d9c.png">
<img width="819" alt="Screen Shot 2023-04-18 at 9 14 45 PM" src="https://user-images.githubusercontent.com/52656433/232839772-8d0cf260-c562-4102-8aee-320c6a04a518.png">
<img width="1754" alt="Screen Shot 2023-04-18 at 9 14 53 PM" src="https://user-images.githubusercontent.com/52656433/232839787-d6208389-b0ce-4703-8adc-5bd3d97dccc4.png">
<img width="1754" alt="Screen Shot 2023-05-03 at 1 42 48 PM" src="https://user-images.githubusercontent.com/52656433/235869833-402aaa6f-15ae-4a37-ab4f-c85bc6359c8e.png">



LAPTOP 1140px:
<img width="1155" alt="Screen Shot 2023-04-18 at 8 52 33 PM" src="https://user-images.githubusercontent.com/52656433/232837146-e262d613-c5d6-4235-b65e-8337dda8e5a9.png">
<img width="1155" alt="Screen Shot 2023-04-18 at 8 52 53 PM" src="https://user-images.githubusercontent.com/52656433/232837238-fb1a115b-2240-455e-afe1-42c784cd3414.png">
<img width="1155" alt="Screen Shot 2023-04-18 at 8 52 57 PM" src="https://user-images.githubusercontent.com/52656433/232837279-91b3ec29-1ea4-45bc-998f-5995699f03fc.png">

LAPTOP 1024px:
<img width="1155" alt="Screen Shot 2023-04-18 at 8 53 13 PM" src="https://user-images.githubusercontent.com/52656433/232837487-2252d454-ed02-4a73-9ae6-c531864faef2.png">
<img width="1155" alt="Screen Shot 2023-04-18 at 8 53 20 PM" src="https://user-images.githubusercontent.com/52656433/232837586-ad83f529-f60b-405e-be08-91148215d1d7.png">
<img width="1155" alt="Screen Shot 2023-04-18 at 8 53 25 PM" src="https://user-images.githubusercontent.com/52656433/232837609-d3fe251b-2b2e-4255-86ab-2946d9c9535a.png">


TABLET:
<img width="773" alt="Screen Shot 2023-04-18 at 8 53 39 PM" src="https://user-images.githubusercontent.com/52656433/232837737-db652100-7f3c-4d3b-add6-b1f6bfa78143.png">
<img width="773" alt="Screen Shot 2023-04-18 at 8 53 45 PM" src="https://user-images.githubusercontent.com/52656433/232837805-9d316b12-3a4d-42f8-abda-014ca997040a.png">
<img width="773" alt="Screen Shot 2023-04-18 at 8 53 49 PM" src="https://user-images.githubusercontent.com/52656433/232837838-84ed9945-f619-4d63-9513-015636e0f753.png">

MOBILE (L):
<img width="429" alt="Screen Shot 2023-04-18 at 9 02 38 PM" src="https://user-images.githubusercontent.com/52656433/232837936-d56c69a6-b056-441b-a247-3ecbe48f79ae.png">
<img width="429" alt="Screen Shot 2023-04-18 at 9 02 51 PM" src="https://user-images.githubusercontent.com/52656433/232837975-a5ad435b-865a-446e-a373-ee0251571d59.png">
